### PR TITLE
Swedish (modern) & Swedish (traditional) translations

### PR DESCRIPTION
**Pull request by shoogle**
There was disagreement among Swedish translators over whether to use modern or traditional terms in the Swedish [`sv`] translation. To resolve this, I've created a new language for the traditional terms, and updated the names of both languages in Preferences to distinguish between them.

Code | Transifex name| Preferences name | Comment
:---|:---|:---|:---
`sv` | Swedish | Svenska (modern) | Our existing Swedish translation
`sv_SE` | Swedish (Sweden) | Svenska (traditionell) | The new translation

There isn't a language code for "Swedish (traditional)", hence `sv_SE` is used instead.

The existing translation was copied and then edited to create the new translation. See [second commit](https://github.com/musescore/MuseScore/pull/22005/commits/3a85bc0a435c2aa9152404ddeb5b734711ba86ea) for the strings that have changed.

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
We should think about which consequences this has for automatic language selection based on system language code (see `LanguagesService::effectiveLanguageCode`). 
With this PR, users whose _system_ language code returns `sv` would get the modern variant, and users with `sv_SE` would get the traditional variant, but we might want to redirect both to the modern variant by default (for example).

**Comment by shoogle** [comment]
@cbjeukendrup, good point. I've updated the PR to prefer the modern translations when using the system locale. Users would have to choose the traditional ones explicitly, if that's what they want to use.

File: share/locale/languages.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     "sl": "Slovenščina",
     "sr": "Srpski",
     "sr_RS" : "Српски",
-    "sv": "Svenska",
+    "sv": "Svenska (modern)",
+    "sv_SE": "Svenska (traditionell)",
```
Comment (Jojo-Schmitz) [inline share/locale/languages.json:63]: Shouldn't the terms 'modern' and 'traditionell' be Swedish terms? Or are they already

Comment (shoogle) [reply share/locale/languages.json:63]: They are already. English is 'tradition<b>al</b>'.

Comment (Jojo-Schmitz) [reply share/locale/languages.json:63]: Ouch, yes

**Review by cbjeukendrup — APPROVED** [review]

**Pull request merged**